SPECS/scossl/scossl.spec

```diff
@@ -0,0 +1,50 @@
+%define debug_package %{nil}
+Summary:        The SymCrypt engine for OpenSSL (SCOSSL) allows the use of OpenSSL with SymCrypt as the provider for core cryptographic operations
+Name:           scossl
+Version:        0.1
+Release:        1%{?dist}
+License:        MIT License
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System/Libraries
+URL:            https://github.com/microsoft/SymCrypt-OpenSSL
+Source0:        %{name}-%{version}.tar.gz
+BuildRequires:  cmake
+BuildRequires:  gcc
+BuildRequires:  make
+BuildRequires:  symcrypt
+
+%description
+The SymCrypt engine for OpenSSL (SCOSSL) allows the use of OpenSSL with SymCrypt as the provider for core cryptographic operations
+
+%prep
+
+%setup -n microsoft-SymCrypt-OpenSSL-ccdc1d9
+
+%build
+ls /usr/lib/
+cp /usr/lib/libsymcrypt.so ./
```

Comment:
These lines shouldn't be necessary; we should just use the libsymcrypt.so in /usr/lib